tests/braille-specs/ja-kantenji.yaml

```diff
@@ -8,6 +8,7 @@ table:
   __assert-match: ja-kantenji.utb
   contraction: no
   dots: 8
+  unicode-range: ucs4
```

Comment:
IIRC the idea of the unicode-range metadata was that it would be added to the query implicitly and not really meant to be specified explicitly. Having it in a query anyway works, but can select an incompatible table (depending on CHARSIZE). The solution is to conditionally exclude tests in the Makefile based on the value of CHARSIZE.

However, because it kinda sucks to have these exclusions in the Makefile, we could also change how unicode-range is interpreted in a query: 
we could change it so that _both_ the explicit _and_ the implicit values need to match (currently an explicit value overrides an implicit value). In addition we would have to change lou_checkyaml so that it does not fail when a unicode-range in the query results in no table being selected.

Reply:
Maybe something for next release...